articles/quickstart/spa/_includes/_calling_api_create_scope.md

```diff
@@ -1,7 +1,11 @@
 ## Add a Scope
 
-When an `access_token` is received after a user authenticates in your app, it is simply an indication that they have proven their identity. By default, however, the `access_token` does not contain any **authorization** information. To properly limit access to your resources based on authorization, you must use [scopes](https://auth0.com/docs/api-auth/tutorials/adoption/api-tokens).
+By default, the access token does not contain any authorization information. To limit access to your resources based on authorization, you must use scopes. Read more about scopes in the  [api tokens documentation](https://auth0.com/docs/api-auth/tutorials/adoption/api-tokens).
 
-In the settings area for your API within the Auth0 dashboard, navigate to the **Scopes** tab. Add any scopes necessary to limit access to your API resources. While the naming of the scopes is at your discretion, a common pattern is to use `<action>:<resource>`. The example here will use a `read:messages` scope.
+In the Auth0 dashboard, in the **APIs** section, click **Scopes**. Add any scopes you need to limit access to your API resources.
```

Comment:
We use `APIs` since that's what the left-hand menu reads. You can add a link using `${manage_url}/#/apis`

(the variable `${manage_url} translates to `https://manage.auth0.com`)